src/main/js/main_html5.js

```diff
@@ -1750,7 +1750,9 @@ require("../../../html5-common/js/utils/environment.js");
      */
     var dequeueSeek = _.bind(function() {
       if (queuedSeekTime === null) { return; }
-      if (this.seek(queuedSeekTime)) { queuedSeekTime = null; }
+      var seekTime = queuedSeekTime;
+      queuedSeekTime = null;
+      this.seek(seekTime);
```

Comment:
Note that if this seek fails due to the lack of a safe seekable range, another seek is queued up again in the seek function.